RobotConstantsBase.java

```diff
@@ -40,6 +40,11 @@ public  static class SwerveDriveBase {
         public static final double kMaxDriveCurrent = 70.0;//Defaults
         public static final double kMaxSteerCurrent = 30.0;//Defaults
 
+        public static final double kCurrentSampleTimeRange = 5;
+        public static final int kCurrentSampleSize = (int) (kCurrentSampleTimeRange * 50); // mustangPeriodic runs 50hz
+        public static final double kTotalDriveCurrentThreshold = 120; // 30 per motor
```

Comment:
We'll have to see how this goes in practice matches and be ready to adjust it. 120 might be low. But let's see